src/otx/recipe/detection/rtdetr_101.yaml

```
@@ -0,0 +1,115 @@
+model:
+  class_path: otx.algo.detection.rtdetr.OTXRtDetr101
+  init_args:
+    label_info: 80
+
+    optimizer:
+      class_path: torch.optim.AdamW
+      init_args:
+        lr: 0.0001
+        betas: [0.9, 0.999]
+        weight_decay: 0.0001
+
+    scheduler:
+      class_path: otx.core.schedulers.LinearWarmupSchedulerCallable
+      init_args:
+        num_warmup_steps: 5
+        main_scheduler_callable:
+          class_path: lightning.pytorch.cli.ReduceLROnPlateau
+          init_args:
+            mode: max
+            factor: 0.1
+            patience: 6
+            monitor: val/map_50
+
+engine:
+  task: DETECTION
+  device: auto
+
+callback_monitor: val/map_50
+
+data: ../_base_/data/torchvision_base.yaml
+overrides:
+  callbacks:
+    - class_path: otx.algo.callbacks.adaptive_train_scheduling.AdaptiveTrainScheduling
+      init_args:
+        max_interval: 1
+        decay: -0.025
+        min_lrschedule_patience: 3
+    - class_path: otx.algo.callbacks.adaptive_early_stopping.EarlyStoppingWithWarmup
+      init_args:
+        monitor: null
+        mode: max
+        patience: 10
+        check_on_train_epoch_end: false
+        min_delta: 0.001
+        warmup_iters: 100
+        warmup_epochs: 7
+  data:
```

```suggestion
        warmup_epochs: 7
        
  data:
```
Could you add empty line between second level groups or (train, val, test) subsets?

---

Done https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/d5b77c306e41641e9616d5d3975f6b84d5e3f390